posthog/celery.py

```diff
@@ -171,6 +171,10 @@ def setup_periodic_tasks(sender: Celery, **kwargs):
             name="count tiles with no filters_hash",
         )
 
+    sender.add_periodic_task(
+        crontab(minute="*"), check_flags_to_rollback.s(), name="check feature flags that should be rolled back"
```

Comment:
Hmm, do we need to still figure out what's going wrong here? Or did you figure it out?

If yes, we should run this less often?

Reply:
Definitely less often